src/main/java/pl/agh/edu/master_diet/core/model/rest/diary/SingleRecentProductResponse.java

```diff
@@ -0,0 +1,33 @@
+package pl.agh.edu.master_diet.core.model.rest.diary;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import pl.agh.edu.master_diet.core.model.shared.MealType;
+import pl.agh.edu.master_diet.core.model.shared.Unit;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SingleRecentProductResponse {
```

Comment:
This class isn't used directly as a response, it's only contained by response model. Maybe it should be renamed to something like `SingleRecentProductInfo`?

Reply:
Done